.travis.yml

```diff
@@ -7,12 +7,21 @@ jobs:
     - stage: Linting
       name: shellcheck
       script: bash -c 'shopt -s globstar; shellcheck **/*.bash **/*.sh'
+
     - &docker-generation
       stage: Docker generation
       language: generic
-      services: docker
       name: tue-env
-      script: ci/build-docker-image.sh tue-env
+      before_install:
+        - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
Maybe better to put this in a script? So we could also use it in other travis file if needed for running the docker containers.

Reply:
Other travis files won't need this because only this is building the image while others are just pulling and executing a pre-built image.

Reply:
Ok, but I still prefer a script.

Reply:
Done.